Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -1490,6 +1488,17 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
     }
   };
 
+  function multiplyNonNullMatrices(keyMat, actMat, inverseShiftScaleMat) {
+    const tmpMat = mat4.clone(keyMat);
```

Comment:
You might want to use model.tmpMat4 to avoid instantiation of a new matrix each time